src/ec/packages/ec-component-footer/demo/data.js

```diff
@@ -1,217 +1,61 @@
-          type: 'branded',
-          name: 'twitter',
-        },
-      },
-      {
-        link: {
-          label: 'Other social networks',
-          path: '/example',
-          icon_position: 'before',
-        },
-        icon: {
-          path: 'static/icons.svg',
-          type: 'ui',
-          name: 'external',
-        },
-      },
-    ],
-  },
-  info: [
-    {
-      link: {
-        label: 'Contact',
-        path: '/example',
-      },
-    },
-    {
-      link: {
-        label: 'Sitemap',
-        path: '/example',
-      },
-    },
-    {
-      link: {
-        label: 'Lorem ipsum',
-        path: '/example',
-      },
-    },
-    {
-      link: {
-        label: 'Lorem ipsum dolor sit',
-        path: '/example',
-      },
-    },
-  ],
+    label: specDataCustom.identity.follow.label,
+    links: specDataCustom.identity.follow.links.map(l => {
+      return formatLink(l);
+    }),
+  },
+  info: specDataCustom.identity.info.map(l => {
+    return formatLink(l);
+  }),
 };
 
-export const sections = [
-  {
-    title: 'European Commission',
-    links: [
-      {
-        link: {
-          label: 'Commission and its priorities',
-          path: '/example',
-        },
-      },
-      {
-        link: {
-          label: 'Policies, information and services',
-          path: '/example',
-        },
-      },
-    ],
-  },
-  {
-    title: 'Follow the European Commission',
-    links: [
-      {
-        link: {
-          label: 'Facebook',
-          path: '/example',
-          icon_position: 'before',
-        },
-        icon: {
-          path: 'static/icons.svg',
-          type: 'branded',
-          name: 'facebook',
-        },
-      },
-      {
-        link: {
-          label: 'Twitter',
-          path: '/example',
-          icon_position: 'before',
-        },
-        icon: {
-          path: 'static/icons.svg',
-          type: 'branded',
-          name: 'twitter',
-        },
-      },
-      {
-        link: {
-          label: 'Other social networks',
-          path: '/example',
-          icon_position: 'before',
-        },
-        icon: {
-          path: 'static/icons.svg',
-          type: 'ui',
-          name: 'external',
-        },
-      },
-    ],
-  },
-  {
-    title: 'European Union',
-    links: [
-      {
-        link: {
-          label: 'European Union',
-          path: '/example',
-          icon_position: 'after',
-        },
-        icon: {
-          path: 'static/icons.svg',
-          type: 'ui',
-          name: 'external',
-        },
-      },
-      {
-        link: {
-          label: 'EU institutions',
-          path: '/example',
-          icon_position: 'after',
-        },
-        icon: {
-          path: 'static/icons.svg',
-          type: 'ui',
-          name: 'external',
-        },
-      },
-    ],
-  },
-];
+export const sections = specDataCorporate.sections.map(s => {
+  return {
+    title: s.title,
+    links: s.links.map(l => {
```

Comment:
Another syntax hint, if you get tired of `return`, `s.links.map(l => formatLink(l))` or even `s.links.map(formatLink)` should be the same.